app/javascript/blacklight/search_context.js

```diff
@@ -15,8 +15,8 @@ Blacklight.doSearchContextBehavior = function() {
   })
 };
 
-Blacklight.csrfToken = () => document.querySelector('meta[name=csrf-token]')?.content
-Blacklight.csrfParam = () => document.querySelector('meta[name=csrf-param]')?.content
+Blacklight.csrfToken = function() { return (document.querySelector('meta[name=csrf-token]') || {}).content }
+Blacklight.csrfParam = function() { return (document.querySelector('meta[name=csrf-param]') || {}).content }
```

Comment:
Now I wonder why Blacklight.csrfToken/csrfParam are defined twice, in two different places, identically? But that pre-dated this PR, so not relevant to approving it, just wondering.

Reply:
One is the compiled distributed code and one is the source code.